docs/user/components/seqsero2_assembly.rst

```diff
@@ -0,0 +1,48 @@
+seqsero2_assembly
+=================
+
+Purpose
+-------
+
+Salmonella serotype determination from genome assemblies.
+
+.. note::
+    Software page: https://github.com/denglab/SeqSero2
+
+Input/Output type
+-----------------
+
+- Input type: ``Fasta``
+- Output type: None
+
+.. note::
+    The default input parameter for fasta data is ``--fasta``.
+
+Published results
+-----------------
+
+- ``results/typing/seqsero2_assembly``: Stores SeqSero2 results for each sample.
+
+Published reports
+-----------------
+
+None.
+
+Default directives
+------------------
+
+- ``cpus``: 1
+- ``memory``: 1GB (dynamically increased with CPUs and on retry)
+- ``container``: ummidock/seqsero2
+- ``version``: alpha-test-1
+- ``cache``: false
```

Comment:
why do you use cache = false? This eliminates the -resume option from nextflow

Reply:
I'll change it